client/Windows/wf_client.c

```diff
@@ -638,6 +638,33 @@ static DWORD wf_verify_changed_certificate_ex(freerdp* instance, const char* hos
 	}
 }
 
+static BOOL wf_present_gateway_message(freerdp* instance, UINT32 type, BOOL isDisplayMandatory,
```

Comment:
Just noticed, is there a documentation of what values `type` can have? An enum here would greatly improve readability (remove the magic numbers)

Reply:
@akallabeth Yes - they are actually already there (in core/gateway/tsg.h) so I will just export them to the public API. 👍

Reply:
@mfleisz great, then please also change the type for the callback accordingly.